homeassistant/components/switch/tuya.py

```diff
@@ -0,0 +1,88 @@
+"""
+Simple platform to control **SOME** Tuya switch devices.
+
+It uses a slightly modified version of the pytuya library
+(https://github.com/clach04/python-tuya) to directly control the switch.
+
+Most switch devices that use the Tuya cloud should work.
+If port 6668 is open on the device then it will work.
+"""
+import voluptuous as vol
+from homeassistant.components.switch import SwitchDevice, PLATFORM_SCHEMA
+from homeassistant.const import (CONF_NAME, CONF_HOST, CONF_ID)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['pytuya==1.0']
+
+CONF_DEVID = 'device_id'
+CONF_LOCKEY = 'local_key'
+
+DEFAULT_ID = 1
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_NAME): cv.string,
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_DEVID): cv.string,
+    vol.Required(CONF_LOCKEY): cv.string,
+    vol.Optional(CONF_ID, default=DEFAULT_ID): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Tuya switch."""
```

Comment:
It should be `"""Set up of the...`. Docstrings should be valid according [PEP257](https://home-assistant.io/developers/development_guidelines/).